samples/msal-node-samples/ElectronSystemBrowserTestApp/redirect/public/JS/handleRedirect.js

```diff
@@ -0,0 +1,17 @@
+"use static";
+
+import auth from "../config/authConfig.json" assert { type: "json" };
+
+window.addEventListener("DOMContentLoaded", function () {
```

Comment:
You're using a template literal below, which is ES6 syntax. Should you change this line to the following to be consistent?:

window.addEventListener("DOMContentLoaded", () => {

Reply:
Changed event to follow ES6 syntax format